From e6449e713f4f5cc90b2445f4bac5332936cec94c Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:04:02 -0400
Subject: [powerpc] eeh: Update MAINTAINERS

Message-id: <1410545655-205645-14-git-send-email-dzickus@redhat.com>
Patchwork-id: 94073
O-Subject: [RHEL7 PATCH 013/626] powerpc/eeh: Update MAINTAINERS
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit ec207dcc91c62aeccb65b36c7764076f5e5aaddb
Author: Gavin Shan <shangw@linux.vnet.ibm.com>
Date:   Fri Jun 28 21:12:14 2013 +0800

    powerpc/eeh: Update MAINTAINERS

    Update MAINTAINERS to reflect recent changes.

    Signed-off-by: Gavin Shan <shangw@linux.vnet.ibm.com>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/MAINTAINERS b/MAINTAINERS
index 5ce636f..310fe3f 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -3114,6 +3114,13 @@ M: Maxim Levitsky <maximlevitsky@gmail.com>
 S: Maintained
 F: drivers/media/rc/ene_ir.*
 
+ENHANCED ERROR HANDLING (EEH)
+M: Gavin Shan <shangw@linux.vnet.ibm.com>
+L: linuxppc-dev@lists.ozlabs.org
+S: Supported
+F: Documentation/powerpc/eeh-pci-error-recovery.txt
+F: arch/powerpc/kernel/eeh*.c
+
 EPSON S1D13XXX FRAMEBUFFER DRIVER
 M: Kristoffer Ericson <kristoffer.ericson@gmail.com>
 S: Maintained
@@ -6192,7 +6199,6 @@ M: Linas Vepstas <linasvepstas@gmail.com>
 L: linux-pci@vger.kernel.org
 S: Supported
 F: Documentation/PCI/pci-error-recovery.txt
-F: Documentation/powerpc/eeh-pci-error-recovery.txt
 
 PCI SUBSYSTEM
 M: Bjorn Helgaas <bhelgaas@google.com>
-- 
1.7.1